### PR TITLE
Fix for #51

### LINKED ISSUE #51: unicode problems
I'm having trouble writing the bibtex file (containing unicode) below. I assume the unicode characters fail to write because `bibtexparser` wants to write ascii, which would work if the unicode would be substituted by the respective latex string. So how do I enforce unicode-to-latex conversion? The correct conversion seems to be in `latexenc.py`: `("\u2009", "\\hspace{0.167em}"),`, but it does not seem to get applied.

Thanks

This code 

```
with open(outfile, 'w') as bibtex_file:
    bibtexparser.dump(bibdb, bibtex_file, writer=writer)
```

causes this error

```
Traceback (most recent call last):
[..]
    bibtex_file.write(bibtex_str)
UnicodeEncodeError: 'ascii' codec can't encode character u'\u2009' in position 76: ordinal not in range(128)
```

on this bibtex file

```
@article{Mesa-Gresa2013,
abstract = {During a 4-week period half the mice (n = 16) were exposed to EE and the other half (n = 16) remained in a standard environment (SE). Aggr. Behav. 9999:XX-XX, 2013. © 2013 Wiley Periodicals, Inc.},
author = {Mesa-Gresa, Patricia and P\'{e}rez-Martinez, Asunci\'{o}n and Redolat, Rosa},
doi = {10.1002/ab.21481},
file = {:Users/jscholz/Documents/mendeley/Mesa-Gresa, P\'{e}rez-Martinez, Redolat - 2013 - Environmental Enrichment Improves Novel Object Recognition and Enhances Agonistic Behavior.pdf:pdf},
issn = {1098-2337},
journal = {Aggressive behavior},
month = apr,
number = {April},
pages = {269--279},
pmid = {23588702},
title = {{Environmental Enrichment Improves Novel Object Recognition and Enhances Agonistic Behavior in Male Mice.}},
url = {http://www.ncbi.nlm.nih.gov/pubmed/23588702},
volume = {39},
year = {2013}
}
```

### PR DESCRIPTION
**Pull request by Phyks**
This should implement the fix described in #51.

Closes #51.

### CODE REVIEW

**Comment by Phyks** [comment]
Hmm, Travis gives a blank page when trying to see the tests results, and the unittests are passing on my local machine.

Also, it seems https://coveralls.io/builds/10804813/source?filename=bibtexparser%2Ftests%2Ftest_bwriter.py#L30 is no longer needed and I could delete it.

**Comment by Phyks** [comment]
Should be fixed.

File: bibtexparser/__init__.py [hunk 1]
```diff
@@ -105,4 +107,8 @@ def dump(bib_database, bibtex_file, writer=None):
     """
     if writer is None:
         writer = bwriter.BibTexWriter()
-    bibtex_file.write(writer.write(bib_database))
+    if sys.version_info >= (3,0):
```
Comment (sciunto) [inline bibtexparser/__init__.py]: PEP8: space after comma.

**Comment by Phyks** [comment]
Should be fixed. There are quite a few of other PEP8 derivations in the codebase, I assigned an issue to me to fix these (https://github.com/sciunto-org/python-bibtexparser/issues/157).

**Comment by sciunto** [comment]
Thanks! It goes in!

**Comment by Phyks** [comment]
Thanks!

**Pull request merged**